### ISSUE TITLE
About the code

### ISSUE THREAD

**Issue by jiang4869** [cardwing/Codes-for-Lane-Detection#187]
I have been reading the source code of etfnet-culane-pytorch for two days, but I cannot find the code of self-attention distillation in which module, could you please tell me?

**Comment by cardwing** [comment]
Please refer to [this issue](https://github.com/cardwing/Codes-for-Lane-Detection/issues/183).

**Comment by harudaee** [comment]
> I have been reading the source code of etfnet-culane-pytorch for two days, but I cannot find the code of self-attention distillation in which module, could you please tell me?

Sir, have you found the code of self-attention distillation in ENet-Label-Torch? I cannot find where they are and figure out how it works.

**Comment by cardwing** [comment]
It is in the model definition and loss function in ENet-Label-Torch. Currently, the mimicking path is block 3 mimics block 4 and block 4 mimics labels.
